hosts/main/Program.cs

```diff
@@ -1,6 +1,7 @@
 // Copyright (c) Duende Software. All rights reserved.
 // See LICENSE in the project root for license information.
 
+using Duende.IdentityServer.Licensing.v2;
```

Comment:
We've put the license services in the v2 namespace to help make sure that changes we make don't introduce regressions. But do we want to show this to our end users? Maybe the internal stuff should be in v2 and the ILicenseSummary should be in Duende.IdentityServer.Licensing?